spec/test_suite_parser.rb

```diff
@@ -5,27 +5,27 @@
 # to run them as rpsec tests
 
 module ProtocolHelper
-  TEST_SUITE_RELATIVE_PATH = ENV['TEST_SUITE_RELATIVE_PATH'] || '../mauth-protocol-test-suite'
-  CASE_PATH = "#{TEST_SUITE_RELATIVE_PATH}/protocols/MWSV2".freeze
+  TEST_SUITE_SUBMODULE_PATH = 'spec/fixtures/mauth-protocol-test-suite'.freeze
```

Comment:
put the magic comment at the top of the file instead of calling `.freeze` here and there?